cypress/integration/sales/sales_order_to_shipment_to_invoice.js

```diff
@@ -82,7 +82,7 @@ describe('Create Sales order', function() {
       .get('li')
       .eq('1')
       .click({ force: true });
-    // cy.wait(8000);
+    cy.wait(8000);
     cy.get('.btn-header.side-panel-toggle').click({ force: true });
```

Comment:
won't adding a timeout option here work ? Not a fan of explicitly waiting for something to happen.

https://docs.cypress.io/api/commands/click.html#Arguments

Reply:
tried,not worked.